fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableIdStmt.java

```diff
@@ -0,0 +1,88 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.analysis;
+
+import com.google.common.base.Strings;
+import org.apache.doris.catalog.Catalog;
+import org.apache.doris.catalog.Column;
+import org.apache.doris.catalog.ScalarType;
+import org.apache.doris.cluster.ClusterNamespace;
+import org.apache.doris.common.AnalysisException;
+import org.apache.doris.common.ErrorCode;
+import org.apache.doris.common.ErrorReport;
+import org.apache.doris.mysql.privilege.PrivPredicate;
+import org.apache.doris.qe.ConnectContext;
+import org.apache.doris.qe.ShowResultSetMetaData;
+
+// SHOW TABLES
+public class ShowTableIdStmt extends ShowStmt {
+    private String db;
+    private long tableId;
+
+    public ShowTableIdStmt(String db, long tableId) {
+        this.db = db;
+        this.tableId = tableId;
+    }
+
+    public String getDb() {
+        return db;
+    }
+
+    public long getTableId() {
+        return tableId;
+    }
+
+    @Override
+    public void analyze(Analyzer analyzer) throws AnalysisException {
+        // check access first
+        if (!Catalog.getCurrentCatalog().getAuth().checkGlobalPriv(ConnectContext.get(), PrivPredicate.ADMIN)) {
+            ErrorReport.reportAnalysisException(ErrorCode.ERR_SPECIFIC_ACCESS_DENIED_ERROR, "SHOW TABLE");
+        }
+        if (Strings.isNullOrEmpty(db)) {
+            db = analyzer.getDefaultDb();
```

Comment:
Do we need to check db here?
I think the purpose of this stmt is to find out which database is this table belongs to.